node/src/command.rs

```diff
@@ -275,6 +258,14 @@ pub fn run() -> sc_cli::Result<()> {
 
             Ok(())
         },
+        Some(Subcommand::Validator(cmd)) => {
+            let runner = cli.create_runner(cmd)?;
+            log::set_max_level(log::LevelFilter::Off);
```

Comment:
We can use _stderr_ as output for warns and errors and _stdout_ for command prints. It doesn't merge when redirecting shell pipes `> >>`.

Reply:
Oh, but it makes sense to not completely mute it, but change the `LevelFilter` to `Error` and then redirect it to `stderr`.

Reply:
> Oh, but it makes sense to not completely mute it, but change the `LevelFilter` to `Error` and then redirect it to `stderr`.

moreover, it wouldn't print the errors from the result of the commands with `LevelFilter::Off`.

Reply:
but then [this](https://github.com/paritytech/substrate/issues/10583#issuecomment-1005090917) happens...

Reply:
Nah, it this case keep it simple as it is now.

Reply:
Yeah, but as I mentioned, in this case it wouldn't print even the errors (i.e. `Result::Err`) from the command itself.

Reply:
Looks like it's time to some tricks.
Maybe we can do something like `.map_err(|e| log::set_level(<_>::Warn); log_error(e); log::set_level(<_>::Off);`, just enable required log level for this particular error?